### DISCUSSION TITLE
Install firefox version .exe

### DISCUSSION THREAD

**Discussion by JRHille** [microsoft/winget-pkgs#99504] [General]
Hello

I'm trying to install Mozilla firefox, but when I type the command **(winget install --id=Mozilla.Firefox -e -h --force --accept-package-agreements --accept-source-agreements --scope "machine")** it installs the version from the store, I would like to install the .exe or .msi, because I need to install it for everyone on the machine, and the command --scope=machine seems not to works with store version.

Would you have any option to choose to install the .exe version?

**Comment by ItzLevvie** [comment]


**Reply by JRHille** [reply]
@ItzLevvie 
Hello,
First thanks for replying.

So, I'm using the same command and here I get this error: It's in my  language (pt-BR), but it says: ''The current system configuration does not support the installation of this package.''

![erro1](https://user-images.githubusercontent.com/114708288/225192341-7da6b4bc-f604-423c-9d6a-28a0465d7e40.png)

I looked in the log and it looks like it's trying to install the .msix installer, so it's giving the error from what I understand.

And if I try to run it without the --scope=machine command it installs the store version. 😢

Here is the log:
[WinGet-2023-03-14-23-41-13.196.log](https://github.com/microsoft/winget-pkgs/files/10975520/WinGet-2023-03-14-23-41-13.196.log)

I also tried adding the **--source winget** command, but it also gives the error.
